framework/src/meshmodifiers/ElementSubdomainModifierBase.C

```diff
@@ -215,11 +216,13 @@ ElementSubdomainModifierBase::modify(
 
   // Initialize solution and stateful material properties
   applyIC(/*displaced=*/false);
+  // if (_fe_problem.getMaterialWarehouse().hasActiveObjects(0))
```

Comment:
```suggestion
```

Reply:
oh this is an accident. this line should be there and uncommented